modules/server/src/main/ts/bedrock/compiler/Webpack.ts

```diff
@@ -296,52 +296,70 @@ export const devserver = async (settings: WebpackServeSettings): Promise<Serve.S
   const tsConfigFile = settings.config;
 
   const compileInfo = await getCompileInfo(tsConfigFile, scratchDir, settings.basedir, true, settings.testfiles, settings.coverage);
-  return Serve.startCustom(settings, (handler) => {
+  return Serve.startCustom(settings, (port, handler) => {
     const scratchFile = compileInfo.scratchFile;
-    console.log(`Loading ${settings.testfiles.length} tests...`);
+    console.log(`Loading ${settings.testfiles.length} test files...`);
```

Comment:
Updated this console message as there's normally a few tests per file so this was misleading.